src/object/srv_obj_migrate.c

```diff
@@ -3957,9 +3959,16 @@ obj_tree_lookup_cont(daos_handle_t toh, uuid_t co_uuid, daos_handle_t *cont_toh)
 		return rc;
 	}
 
+	memset(&uma, 0, sizeof(uma));
+	uma.uma_id = UMEM_CLASS_VMEM;
 	cont_root = tmp_iov.iov_buf;
-	*cont_toh = cont_root->tcr_root_hdl;
-	return 0;
+	rc = dbtree_open_inplace(&cont_root->tcr_btr_root, &uma, &migrated_toh);
```

Comment:
Do we need to check `if (daos_handle_is_inval(&cont_root->tcr_root_hdl))` before open?

Reply:
Since the above code at L3952 the container lookup succeed, it means the container must with OID being migrated, then the container's migrated obj tree should have already been created.
Can add an assertion the handle is valid although, I'll add it.